scripts/python/testcases/tcTrigger.py

```diff
@@ -0,0 +1,253 @@
+import api.tutils
+import time
+import sys
+import api.objmodel
+import random
+
+# Check ping between all containers in the same network
+def checkPingContainersInNetworks(containers, networks):
+    cntrIpList = []
+    # Read all IP addresses
+    for cnt in containers:
+        cntrIp = cnt.getIpAddr()
+        cntrIpList.append(cntrIp)
+
+    nidx = random.randint(0, (len(networks) - 1))
```

Comment:
my general concern with random testing is that it can soon become a sanity nightmare if there is no where to reproduce an issue and verify fix. A test environment should be repeatable as much as possible. If I am reading this correct, it doesn't seem possible to repeat a test run with the current approach?